ruff.toml

```diff
@@ -1,3 +1,5 @@
+include = ["examples/**/*.py", "sentence_transformers/**/*.py", "tests/**/*.py", "docs/**/*.py", "setup.py"]
```

Comment:
Which ruff-checkable/formattable files are excluded from these? I'm wondering if this explicit setting can be dropped to keep things simple

Reply:
I think None, it seems plausible to drop this completely. I think this setup is just leftover from another project of mine, where some files were purposefully excluded.